site/content/en/docs/Tutorials/_index.md

```diff
@@ -5,12 +5,7 @@ linkTitle: "Tutorials"
 weight: 8
 date: 2017-01-04
 description: >
-  Show your user how to work through some end to end examples.
+  Contributed end-to-end tutorials using minikube
 ---
 
-{{% pageinfo %}}
-This is a placeholder page that shows you how to use this template site.
-{{% /pageinfo %}}
-
-Tutorials are **complete worked examples** made up of **multiple tasks** that guide the user through a relatively simple but realistic scenario: building an application that uses some of your project’s features, for example. If you have already created some Examples for your project you can base Tutorials on them. This section is **optional**. However, remember that although you may not need this section at first, having tutorials can be useful to help your users engage with your example code, especially if there are aspects that need more explanation than you can easily provide in code comments.
-
+Tutorials are **complete worked examples** made up of **multiple tasks** that guide the user through a relatively simple but realistic scenario: building an application that uses some of your project’s features, for example. If you have already created some Examples for your project you can base Tutorials on them. 
```

Comment:
Should we delete this boilerplate?

Reply:
I've decided to leave this notice for now as guidance to future tutorial authors. It seems useful.